src/components/Button.js

```diff
@@ -30,3 +27,17 @@ const StandardButton = styled.button`
     transform: scale(0.95);
   }
 `;
+
+export default Button;
+
+const ButtonPrimary = styled(Button)`
+  background-color: var(--bg-color-primary);
+`;
+
+export { ButtonPrimary };
```

Comment:
I am not sure: Are the {} necessary here?

Reply:
It doesn't work because it's not the default export of the button component.